src/components/Onboarding/Screens/Apps/ConvictionVotingSettings/AdvancedSettingsModal.js

```diff
@@ -0,0 +1,60 @@
+import { Help } from '@1hive/1hive-ui'
```

Comment:
We should include an optional field to configure a custom `requestToken`. Do not worrie too much about it, let's include it in a future PR.